airflow/providers/exasol/hooks/exasol.py

```diff
@@ -143,7 +143,9 @@ def export_to_file(
             )
         self.log.info("Data saved to %s", filename)
 
-    def run(self, sql: Union[str, list], autocommit: bool = False, parameters: Optional[dict] = None) -> None:
+    def run(
+        self, sql: Union[str, list], autocommit: bool = False, parameters: Optional[dict] = None, handler=None
```

Comment:
Let’s make `sql` a `Union[str, List[str]]`.

Why is `handler` added?

Reply:
The error here is because of signature not matching the run() in DbApiHook
```
mypy --namespace-packages airflow/providers/exasol/hooks
airflow/providers/exasol/hooks/exasol.py:146: error: Signature of "run" incompatible with supertype "DbApiHook"
        def run(
        ^
```